modules/xmpp/SdpConsistency.js

```diff
@@ -78,6 +38,9 @@ export default class SdpConsistency {
      *  in future calls to makeVideoPrimarySsrcsConsistent
      */
     setPrimarySsrc (primarySsrc) {
+        if (typeof primarySsrc !== 'number') {
```

Comment:
My intention here is to crash the code instead of handling two different types (string or number)